ricecooker/classes/nodes.py

```diff
@@ -454,7 +458,11 @@ def __init__(self, id, title, files, author="", description="", license=None, ex
         files = [] if files is None else files
 
         # Set mastery model defaults if none provided
-        exercise_data = {'mastery_model': exercises.M_OF_N, 'randomize': True, 'm': 3, 'n': 5} if exercise_data is None else exercise_data
+        exercise_data = {} if exercise_data is None else exercise_data
```

Comment:
Enforcing mastery_model on exercise nodes: https://trello.com/c/YJnX5kB4/244-ricecooker-does-not-enforce-mastery-model-on-exercises-but-export-process-does-so-magogenie-channel-cannot-be-exported